packages/format-library/src/bold/index.js

```diff
@@ -13,8 +13,15 @@ export const bold = {
 	title,
 	tagName: 'strong',
 	className: null,
-	edit( { isActive, value, onChange } ) {
-		const onToggle = () => onChange( toggleFormat( value, { type: name } ) );
+	edit( { isActive, value, onChange, onFocus } ) {
+		function onToggle() {
+			onChange( toggleFormat( value, { type: name } ) );
+		}
+
+		function onClick() {
+			onToggle();
+			onFocus();
```

Comment:
This change is something that will impact third-party developers creating their own formats, I guess we should refer this PR in a dev note.

Reply:
Added the label. I won't break anything, but yes it's a change in behaviour. :)